data/mappings/info_config.json

```diff
@@ -92,6 +103,7 @@
     "SECURE_IDLE_TIMEOUT": {"info_key": "secure.idle_timeout", "value_type": "int"},
     "SENDSTRING_BELL": {"info_key": "audio.macro_beep", "value_type": "bool"},
     "SPLIT_MODS_ENABLE": {"info_key": "split.transport.sync_modifiers", "value_type": "bool"},
+    "SPLIT_POINTING_ENABLE": {"info_key": "split.pointing_device.enabled", "value_type": "bool"},
```

Comment:
I expected this to fit into the `transport` options - or how would you distinguish these here?

Reply:
ah, fair.
```suggestion
    "SPLIT_POINTING_ENABLE": {"info_key": "split.transport.sync_pointing", "value_type": "bool"},
```

Reply:
`split.transport.sync_pointing_device` ? 
though I'm not sure what is actually synced - but all other options use `_DEVICE_` as part of the name.

Reply:
Honestly, I'm horrible at naming...

Not sure if it's better to use the full "pointing device" as per the other features, or keep it closer to what the other transport.sync_* names are for consistency (with both naming in the info.json and config.h stuff)